### ISSUE TITLE
Default pagination

### ISSUE THREAD

**Issue by llars0689** [filegator/filegator#198]
Is it possible to tweak the webGUI to disable "No pagination" from the drop-down menu and force it to always page?

The directories that we have on our server can have 30,000+ exr/dpx files and if a user chooses "No pagination" those directories crash the server, since they are way to large to open. It works great with it set to 15 per page and would be great if its possible to tweak the code to only allow paging. If its possible could someone steer me in the right direction on what file to edit this?

Thanks

**Comment by alcalbg** [comment]
Hi,

Pagination is just a front-end convenience, the whole content of the current directory will always be loaded in one go. This is how the underlying flysystem works since most storage adapters don't support reading with offset AFAIK.

See #21 on how to change defaults.

**Comment by llars0689** [comment]
Hi alcalbg,

Thanks for the info. I followed the link in #21 but it seems that my installation doesnt have a "frontend" folder, i tried searching manually for the Browser.vue file, but it doesnt seem to exist. Did a path or filename change in the latest version?

**Comment by llars0689** [comment]
Ah.. i just realised im using the precompiled build, not the development version. So its using the html,css, and javascript frontend. Dont supposed there is a way to hardcode it with this version?

**Comment by alcalbg** [comment]
No, but I'll tag this as a suggestion and try to include some kind of config parameter in future versions.
